internal/git/git.go

```diff
@@ -393,6 +393,40 @@ func (s *Service) Commit(ctx context.Context, rootPath, changesPath, authorName,
 	return gitPush(ctx, rootPath)
 }
 
+func (s *Service) SignedCommit(ctx context.Context, rootPath, changesPath, authorName, authorEmail, msg string, committer Committer) error {
+	span, ctx := s.Tracer.FromCtx(ctx, "git.Commit")
+	defer span.Finish()
+
+	span, _ = s.Tracer.FromCtx(ctx, "add changes")
+	err := execCommand(ctx, rootPath, "git", "add", ".")
+	span.Finish()
+	if err != nil {
+		return errors.WithMessage(err, "add changes")
+	}
+
+	span, _ = s.Tracer.FromCtx(ctx, "check for changes")
+	err = checkStatus(ctx, rootPath)
+	span.Finish()
+	if err != nil {
+		return errors.WithMessage(err, "check for changes")
+	}
+	fullCommitMsg := fmt.Sprintf("%s\nSigned-off-by: %s <%s>", msg, authorName, authorEmail)
```

Comment:
Have you tried to do an artifact release from an artifact created with this commit message?

We should make sure the locate conditions will still detect the releases properly, e.g. `locateReleaseCondition` and `locateServiceReleaseCondition`.